babel.config.js

```diff
@@ -29,26 +29,34 @@ const minimist = require('minimist');
 const {isTravisBuild} = require('./build-system/travis');
 const argv = minimist(process.argv.slice(2));
 
+const isDist = argv._.includes('dist');
+const {esm} = argv;
```

Comment:
Can you send out a follow up PR that documents the `--esm` flag here?

https://github.com/ampproject/amphtml/blob/d879209ea2a00a8ac15b01dedaf11eb8a6661e6a/build-system/tasks/dist.js#L441-L459